### ISSUE TITLE
Wordpress php7.2 path error

### ISSUE THREAD

**Issue by andyjsh** [GoogleCloudPlatform/php-docs-samples#696]
I am trying to do a fresh install of wordpress following the instructions per https://github.com/GoogleCloudPlatform/php-docs-samples/tree/master/appengine/php72/wordpress

Setup steps 1 & 2 work no problems

On php wordpress.php create I am presented with the following errors:

```
C:\Users\name\Documents\Code\company\website2\php-docs-samples\appengine\php72\wordpress>php wordpress.php create
Please enter a directory path for the new project (defaults to my-wordpress-project):
We will use the directory: my-wordpress-project. If the directory exists, we will overwrite the contents. Do you want to continue? (Y/n)y
Downloading the WordPress archive...
PHP Warning:  PharData::extractTo(): No such file or directory in C:\Users\name\Documents\Code\company\website2\php-docs-samples\appengine\php72\wordpress\vendor\google\cloud-tools\src\Utils\Project.php on line 86

In Project.php line 86:

  Unable to create path "\C:\Users\name\AppData\Local\Temp" for extraction
```

Looks like the \ before C is creating the issue maybe?

**Comment by bshaffer** [comment]
Oh wow, this is interesting. This is UNIX bias at it's finest, folks.

I've fixed this in https://github.com/GoogleCloudPlatform/php-tools/pull/47/files and will tag a new version shortly so you can use the sample as expected on Windows.

**Comment by andyjsh** [comment]
Thanks heaps @bshaffer, it looks like the latest update has resolved the error.

However I am getting the following error (doesn't seem to be a showstopper though): 

`PHP Warning:  rename(C:\Users\user\AppData\Local\Temp\wordpress,C:\tmp\php-docs-samples\appengine\php72\wordpress\my-wordpress-project): Access is denied. (code: 5) in C:\tmp\php-docs-samples\appengine\php72\wordpress\vendor\google\cloud-tools\src\Utils\WordPressProject.php on line 161`

Thanks

**Comment by bshaffer** [comment]
This looks similar to #698, which describes a workaround. I will look into it!

**Comment by bshaffer** [comment]
I've added a branch to this repository that hopefully fixes this problem! Would you be willing to give it a try? If so, do the following on your CLI 

```sh
# CD into where you downloaded this repository
cd /path/to/php-docs-samples
# CD into the wordpress sample directory
cd appengine/php72/wordpress
# checkout the branch where I've added the fix
git checkout fix-wordpress-rename
# pull down the new composer shizzle
composer install
# Now try to run the create command again!
php wordpress.php create
```

Let me know if this works better for you, or if you still get the error!

**Comment by bshaffer** [comment]
This will be fixed in #702

**Comment by bshaffer** [comment]
No problem! I am closing this issue for now, but please reopen this issue or open a new issue if you have any other problems!
